docs/management/connectors/action-types/slack.asciidoc

```diff
@@ -69,16 +80,26 @@ Use the <<action-settings,Action configuration settings>> to customize connector
 
 [float]
 [[configuring-slack]]
-==== Configure a Slack account
+=== Configure a Slack account
 
-You need a https://api.slack.com/incoming-webhooks[Slack webhook URL] to 
-configure a Slack account. To create a webhook
-URL, set up an an **Incoming Webhook Integration** through the Slack console:
+Before you can create a Slack connector, you must configure your account and obtain the necessary URL or token.
+
+[float]
+[[configuring-slack-webhook]]
+==== Configure a Slack account with Webhook
 
 . Log in to http://slack.com[slack.com] as a team administrator.
-. Go to https://my.slack.com/services/new/incoming-webhook.
-. Select a default channel for the integration.
-+
-image::images/slack-add-webhook-integration.png[]
-. Click *Add Incoming Webhook Integration*.
+. Follow first 3 paragraphs of https://api.slack.com/messaging/webhooks.
```

Comment:
In case that page changes and the first three paragraphs are not necessarily the same, I recommend that we call out the specific steps required. For example:
```suggestion
. Create a Slack app, enable incoming webhooks, then create an incoming webhook. Refer to https://api.slack.com/messaging/webhooks.
```